docs/example-scenario/apps/fully-managed-secure-apps.md

```diff
@@ -0,0 +1,192 @@
+8. The build VM has the Azure DevOps Agent installed which allows the build VM to pull the bits for the Web App to deploy to the ASE (as the Build VM is deployed in a Subnet inside the same VNET)
+
+### Components
+
+* The [App Service Environment][intro-to-app-svc-env] provides a fully isolated and dedicated environment for securely running the application at high scale. In addition, since ASE and the workloads that run on it are behind a VNET, this provides an additional layer of security and isolation. The requirement of high scale and isolation drove towards selecting ILB ASE.
+* This workload is using the [isolated App Service pricing tier][isolated-tier-pricing-and-ase-pricing] so the application is running in a private, dedicated environment in an Azure datacenter using Dv2-series VMs with faster processors, SSD storage, and double the memory-to-core ratio compared to Standard
+* Azure App Services [Web App][docs-webapps] and [API App][docs-apiapps] hosts web applications and RESTful APIs. These are hosted on the Isolated Pricing Tier plan which also offers auto-scaling, custom domains and so on, but in a dedicated tier.
+* Azure [Application Gateway][docs-appgw] is a web traffic load balancer oerating at Layer 7 that manages traffic to the web application. It offers SSL off-loading, that removes additional overhead on the web servers hosting the web app to decrypt traffic again.  
+* [Web Application Firewall][docs-waf] Web application firewall (WAF) is a feature of Application Gateway. The reason for enabling the WAF in the Application Gateway is to further enhance security. The WAF uses OWASP rules to further protect the web application against attacks such as cross-site scripting, session hijacks, and SQL injection.
+* [Azure SQL Database][docs-sql-database] Azure SQL Database was selected as majority of data in this application is relational data, with some data as documents and BLOB. 
+* [Azure Networking] Azure provides a variety of networking capabilities in Azure and the the netowrks can further be peered with other VNETS in Azure or connectivity can be established with on-prem data centers via Express Route or Site to Site. In this case, the Microsoft.Sql service endpoint is enabled on the VNET, then the VNET is allowed access in the Firewall settings hosting the Azure SQL Database. This was done to ensure the data is flowing just between the Azure VNET and the SQL Database instance.
+* [Azure DevOps][docs-azure-devops] Azure DevOps was used for teams to collaborate during many sprints, utilizing features of Azure DevOps that support Agile Development, and create build and release pipelines. 
+* [Build Azure VM][docs-azure-vm] In Azure Portal, create a Visual Studio Enterprise 2017 (latest release) on Windows Server 2016 (x64) VM that will be used as the build agent. This was done, so that the installed agent can pull down the respective build, and deploy the web app to the ASE environment.
```

Comment:
"In Azure Portal" ... we don't need to provide deployment instructions here. You can just say the build VM uses a Windows image with VS 2017 or something like that. Also "Build Azure VM" doesn't doesn't right.